pkg/db-repo/repository.go

```diff
@@ -27,12 +27,12 @@ const (
 var operations = []string{Create, Read, Update, Delete, Query}
 
 type RecordNotFoundError struct {
-	Id      uint
-	ModelId string
+	Id      *uint
```

Comment:
I think I would make these fields private, if you need to access them, I would provide accessor methods for them.